docs/tutorial/wsl.md

```diff
@@ -90,21 +81,13 @@ apt-get update
 
 with this the local Windows and WSL are all installed. To check `usbipd` tool is working well on both side, please follow the following steps:
 
-1. <span id="usbipd_instructions"></span>open PowerShell command prompt with administrator right and then type in the command `usbipd wsl list`:
-
-   <img src="../../media\tutorials\using_docker_container\usbipd_wsl_l.png" alt="" height="">
-
-   as you can see, all USB devices from Windows have been found and not attached sate.
-
-2. to access the specified device from local Windows on WSL, it needs to bind this device. Open PowerShell command prompt with administrator rights and then type in the command `usbipd bind -b <BUSID>`:
-
-   <img src="../../media\tutorials\using_docker_container\usbipd_bind.png" alt="" height="">
+1. <span id="usbipd_instructions"></span>open PowerShell command prompt with administrator right and then type in the command `usbipd list` for a list of USB serial devices.
 
-   **Note**: this command needs to be used only one time,unless the computer has restarted. **1-1** is the device's bus id I would like to bind.
+2. to access the specified device from local Windows on WSL, it needs to bind this device. Open PowerShell command prompt with administrator rights and then type in the command `usbipd bind --busid <BUSID>`:
```

Comment:
same as above:

to access the specified device from Windows on WSL locally, the device must be bound.